lib/graphql/define/defined_object_proxy.rb

```diff
@@ -10,6 +10,11 @@ def types
         GraphQL::Define::TypeDefiner.instance
       end
 
+      # TODO: do I actually want to add this?
```

Comment:
Could you think of other use cases where `metadata` can be helpful in the future?